app/views/shared/_parole_info.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= parole_record = @prisoner.current_parole_record%>
```

Comment:
Small niggle - if you don't want to actually OUTPUT parole_record, you should just do this (also note the whitespace 😸):

```
<% parole_record = @prisoner.current_parole_record %>
```

`<%=` will output the last returned/evaluated value

Reply:
Ah, my mistake, thank you!